package.json

```
@@ -0,0 +1,85 @@
+{
+  "name": "fashion24-back-end",
+  "version": "0.0.1",
+  "description": "",
+  "author": "team fashion-24 <https://github.com/fashion24-developer>",
+  "private": true,
+  "license": "MIT",
+  "contributors": [
+    "Biho Jeong <jjb26433@naver.com>",
+    "Junhyuk Park <pjh_2004@naver.com>"
+  ],
```

```suggestion
  "contributors": [
    "Biho Jeong <jjb26433@naver.com>",
    "Junhyuk Park <pjh_2004@naver.com>",
    "Young-eun Jo <jo1814@naver.com>"
  ],
```
"Young-eun Jo <jo1814@naver.com>"
입니당

---

```suggestion
  "contributors": [
    "Biho Jeong <jjb26433@naver.com>",
    "Junhyuk Park <pjh_2004@naver.com>",
    "Young-eun Jo <jo1814@naver.com>",
    "Seungwoo LEE <swlee456@naver.com>"
  ],
```